src/main/java/core/basesyntax/strategy/FruitAdder.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+
+public class FruitAdder implements FruitHandler {
+    @Override
+    public void handle(String[] line) {
+        Storage.getStorage().merge(line[INDEX_FRUIT],
```

Comment:
it's a bad practice to communicate with the database from such classes directly, it's better to create a FruitDao interface that will have the needed methods to put some fruit in storage or get all of them